### DISCUSSION TITLE
HW1 - about accuracy

### DISCUSSION THREAD

**Discussion by gkssk3654** [seominjoon/kaist-ai605#10] [Q&A] [answered]
What is the accuracy of the model to be able to learn properly without RNN, LSTM? 
I wonder if the low accuracy is because without RNN, or because I made a wrong model.

**Comment by seominjoon** [comment, marked as answer]
A random model will achieve 50%-ish so your model should be higher than that fore sure.
It depends on your implementation details but LSTM with dropout can be higher than 80% so you should aim for it.

**Reply by youngwanLEE** [reply]
@seominjoon 
Hi, I have a question related to this discussion.

Is it assuming that the LSTM model(higher than 80%) also uses embedding_dimension=3?

**Reply by seominjoon** [reply]
No, please feel free to use higher dim, and I recommend using embedding dim at least 50.

**Reply by youngwanLEE** [reply]
@seominjoon thanks!
